dropwizard-app/src/main/java/org/coner/boundary/RegistrationBoundary.java

```diff
@@ -17,6 +17,10 @@ public static RegistrationBoundary getInstance() {
         return instance;
     }
 
+    public static void setInstance(RegistrationBoundary rb) {
```

Comment:
Is there a reason you're diverging from the convention of parameter name being a lower camel case of the type name? IntelliJ has a shortcut to generate it, which is super helpful for the really long ones. I have mine set to the Eclipe keymap, and to generate the verbose name I would type `r` `B` `Ctrl + Space` `Enter`

</nitpick>